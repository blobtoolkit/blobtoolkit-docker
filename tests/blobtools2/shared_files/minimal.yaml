assembly:
  level: scaffold
  prefix: minimal
taxon:
  name: Genus species