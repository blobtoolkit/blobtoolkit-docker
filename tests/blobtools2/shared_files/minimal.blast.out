seq1	771870	10	seq1	XM_003350831.1	90	10	1	0	1	10	364	373	0.0
seq1	771870	10	seq1	XM_003350817.1	90	10	1	0	1	10	698	3163	0.0
seq1	771870	8	seq1	XM_003350855.1	80	8	0	0	1	8	2116	1	0.0
seq1	771870	6	seq1	XM_003347996.1	50	8	3	0	1	8	549	3160	0.0
seq1	771870	4	seq1	XM_003350829.1	30	4	1	0	1	4	1897	302	0.0
seq2	771870	10	seq2	XM_003350805.1	80	10	2	0	1	10	1982	194	0.0
seq2	771870	9	seq2	XM_003347991.1	80	8	0	0	1	8	132	2128	0.0
seq2	771870	7	seq2	XM_003350856.1	70	7	1	0	1	7	2751	207	0.0
seq2	771870	7	seq2	XM_003347989.1	60	6	1	0	1	6	1826	57	0.0
seq2	771870	6	seq2	XM_003350826.1	50	6	1	0	1	6	1385	109	0.0
